apps/etterlatte-behandling/src/main/kotlin/oppgaveny/OppgaveServiceNy.kt

```diff
@@ -93,16 +93,30 @@ class OppgaveServiceNy(private val oppgaveDaoNy: OppgaveDaoNy, private val sakDa
     }
 
     fun haandterFattetvedtak(attesteringsoppgave: AttesteringsOppgave): OppgaveNy {
-        val behandlingsoppgave = oppgaveDaoNy.hentOppgaveForBehandling(attesteringsoppgave.referanse)
-        if (behandlingsoppgave == null) {
-            throw RuntimeException("Må ha en oppgave fra før av for å kunne lage attesteringsoppgave")
+        val behandlingsoppgaver = oppgaveDaoNy.hentOppgaverForBehandling(attesteringsoppgave.referanse)
+        if (behandlingsoppgaver.isEmpty()) {
+            throw BadRequestException("Må ha en oppgave for å kunne lage attesteringsoppgave")
         } else {
-            oppgaveDaoNy.endreStatusPaaOppgave(behandlingsoppgave.id, Status.FERDIGSTILT)
-            return opprettNyOppgaveMedSakOgReferanse(
-                attesteringsoppgave.referanse,
-                attesteringsoppgave.sakId,
-                attesteringsoppgave.oppgaveType
-            )
+            val oppgaverUnderbehandling = behandlingsoppgaver.filter { it.status == Status.UNDER_BEHANDLING }
```

Comment:
Kunne evt. bruke `behandlingsoppgaver.single { it.status == Status.UNDER_BEHANDLING }`, som kaster NoSuchElementException / IllegalArgumentException avhengig av om det er ingen eller > 1 elementer

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1849